schema/plugin.json

```diff
@@ -0,0 +1,152 @@
+{
+  "title": "Theme editor",
+  "description": "Settings of Theme Editor.",
+  "properties": {
```

Comment:
You should store all CSS properties as in subdictionnary. And add a new settings, to allow the user to say if he prefers to use the value from the theme or from the settings.

This `properties` field should look like:

```json5
"properties": {
  "customStyles": {
    "type": "object",
    "properties": {
      "layout-color": {
      // ...
      },
      // ...
    },
    "additionalProperties": false
  },
  "useTheme": {
    "title": "Use the theme styles",
    "description": "Whether to use the theme styles or the one from the theme editor."
    "type": "boolean",
    "default": true
  }
}
```

Then when starting the extension, you read the value from the settings or from the theme depending on the value of the settings `useTheme`.

Reply:
Thanks for the comment. I have changed the schema but for the moment what I am failing in making the extension work with a boolean enabling to switch between the 2 modes.